draft-ietf-sedate-datetime-extended.md

```diff
@@ -274,27 +275,33 @@ IANA Time Zone:
   called `tz` or `zoneinfo`) maintained by IANA {{TZDB}}{{BCP175}}.
   Most IANA time zones
   are named for the largest city in a particular region that shares
-  the same time zone rules, e.g. `Europe/Paris` or `Asia/Tokyo` {{TZDB-NAMING}}.
+  the same time zone rules, e.g., `Europe/Paris` or `Asia/Tokyo` {{TZDB-NAMING}}.
 
-  Note that the rules defined for a named IANA time zone can change
+  The rules defined for a named IANA time zone can change
   over time.
   The use of a named IANA time zone implies that the intent is for the
-  rules that are current at the time of interpretation to apply, i.e.,
+  rules to apply that are current at the time of interpretation, i.e.,
   the additional information conveyed by using that time zone name is
   to change with the changed rules as recorded in the IANA time zone
   database.
```

Comment:
I found this long sentence to be a bit confusing. Here's a minor suggestion to break it up into two sentences. 

```suggestion
  rules to apply that were current at the time of interpretation.
  If rules in the IANA time zone database are updated, then the
  additional information conveyed by the time zone name may change.
```

Reply:
I think this changes the semantics a bit much.  I made a simpler simplification.